bc_obps/registration/tests/endpoints/v2/_operations/_operation_id/test_bcghg_id.py

```diff
@@ -20,20 +20,19 @@ def test_bcghg_id_endpoint_unauthorized_roles_cannot_patch(self):
             assert response.status_code == 401
             assert response.json()['detail'] == "Unauthorized"
 
-    # def test_authorized_role_can_patch(self):
-    #     roles = ["cas_admin", "cas_analyst"]
-    #     for role in roles:
-    #         response = TestUtils.mock_patch_with_auth_role(
-    #             self,
-    #             role,
-    #             self.content_type,
-    #             {},
-    #             custom_reverse_lazy(
-    #                 "operation_bcghg_id",
-    #                 kwargs={
-    #                     'operation_id': baker.make_recipe('utils.operation')
-    #                 },
-    #             ),
-    #         )
-    #         assert response.status_code == 200
-    #         assert response.json()['id'] is ???
+    def test_authorized_role_can_issue_id(self):
+        operation = baker.make_recipe('utils.operation')
+        roles = ["cas_admin", "cas_analyst"]
+        for role in roles:
+            response = TestUtils.mock_patch_with_auth_role(
+                self,
+                role,
+                self.content_type,
+                {},
+                custom_reverse_lazy(
+                    "operation_bcghg_id",
+                    kwargs={'operation_id': operation.id},
+                ),
+            )
+            assert response.status_code == 200
+            assert response.json()['id'] == '14862100001'  # '1' and '486210' come from the recipe's mock data
```

Comment:
We no longer need to test roles. 🎉 We can repurpose this test to validate the response schema instead.